src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1512,6 +1514,8 @@ public static bool HasNames(this TupleExpressionSyntax tuple)
         {
             // cases:
             //  const var
+            //  ref var
+            //  ref readonly var
```

Comment:
these comment should move down to the new block you added.  right?

Reply:
actually all cases are on the top here, see the rest of the comment - it's all below code i added

Reply:
or do you want me to move each comment to the relevant place where it's handled?

Reply:
yes.

Reply:
i'm not sure if i should make code style changes like that unrelated to what i'm doing and mess with half of this method by moving comments

Reply:
that's fair.  do as you wish.